CHANGES.md

```diff
@@ -19,6 +19,34 @@
 
 # Parquet #
 
+### Version 2.7.0 ###
+
+#### Bug
+
+*   [PARQUET-1437](https://issues.apache.org/jira/browse/PARQUET-1437) - Misleading comment in parquet.thrift
+*   [PARQUET-1554](https://issues.apache.org/jira/browse/PARQUET-1554) - Compilation error when upgrading Scrooge version
+*   [PARQUET-1561](https://issues.apache.org/jira/browse/PARQUET-1561) - Inconsistencies in the Parquet Delta Encoding specification
+
+#### Improvement
+
+*   [PARQUET-1487](https://issues.apache.org/jira/browse/PARQUET-1487) - Do not write original type for timezone-agnostic timestamps
+*   [PARQUET-1539](https://issues.apache.org/jira/browse/PARQUET-1539) - Clarify CRC checksum in page header
+*   [PARQUET-1589](https://issues.apache.org/jira/browse/PARQUET-1589) - Bump Java to 8
+*   [PARQUET-1588](https://issues.apache.org/jira/browse/PARQUET-1588) - Bump Apache Thrift to 0.12.0 in parquet-format
+*   [PARQUET-1590](https://issues.apache.org/jira/browse/PARQUET-1590) - Add Java 11 to Travis
+*   [PARQUET-1610](https://issues.apache.org/jira/browse/PARQUET-1610) - Small spelling issues
+
+#### Task
+
+*   [PARQUET-1572](https://issues.apache.org/jira/browse/PARQUET-1572) - Clarify the definition of timestamp types
+*   [PARQUET-1585](https://issues.apache.org/jira/browse/PARQUET-1585) - Update old external links in the code base
+*   [PARQUET-1627](https://issues.apache.org/jira/browse/PARQUET-1627) - Update specification so that legacy timestamp logical types can be written for local semantics as well
+
+#### New Feature
+
+*   [PARQUET-41](https://issues.apache.org/jira/browse/PARQUET-1630) - Add bloom filters to parquet statistics
```

Comment:
```suggestion
*   [PARQUET-1630](https://issues.apache.org/jira/browse/PARQUET-1630) - Add bloom filters to parquet statistics
```
The Parquet-41 is the umbrella Jira, I think we should mention `PARQUET-1630` here instead.

Reply:
@Fokko should I change the Jira title as well?

Reply:
One of the git commit headlines contains PARQUET-41 so, I would mention it in the release notes.

Reply:
> The Parquet-41 is the umbrella Jira, I think we should mention `PARQUET-1630` here instead.

Maybe nit. Presumably, the titles like "New Feature" are auto-generated from the jira types. So if 1630 ("sub-task") is mentioned instead of 41 ("new feature"), there won't be "new features" in the format 2.7.0 release. When in fact we have two - bloom filter and modular encryption. If these titles don't matter much in changes.md, then no problem, please ignore this. If they matter, then I guess the changes.md can be manually edited to handle the titles - or alternatively, we can create new jiras of the "new feature" type for both bloom filter and encryption, that will go into the 1608.

Reply:
We mainly track all the changes in jira. CHANGES.md is only a generated list of the jiras made into the release. I think, it is useful to have that list separated so one don't have to query the jira for a release but have the list in one place.
We might have additional jiras in this list which are not listed in the git (because of umbrella jiras, subtasks etc.) but I think it is good to have all the headlines in the git being listed here (and of course properly documented in the jira).

Reply:
@gszadovszky Sounds good to me, added format-2.7.0 to PARQUET-1178. Please include it in PARQUET-1608.

Reply:
Done. @gszadovszky , can you have another look? Thanks!